src/Illuminate/Auth/Console/stubs/make/controllers/HomeController.stub

```diff
@@ -14,7 +14,6 @@ class HomeController extends Controller
      */
     public function __construct()
     {
-        $this->middleware('auth');
```

Comment:
Sorry, just noticed it has to stay to handle new methods added to the controller.
Maybe we should remove the middleware from the route group, and add it to the routes in `Route::auth()`. What do you think?